File: wk1/reduplikation.java
// problem link: https://open.kattis.com/problems/reduplikation

import java.io.*;
import java.util.*;
public class reduplikation {
	public static void main(String[] args) throws IOException {
		Scanner c = new Scanner(System.in);
		String s = c.next();
		int a = c.nextInt();
		for(int i = 0; i < a; i++)
			System.out.print(s);
		c.close();
	}
}

File: wk3/cpp/reduplikation.cpp
// problem link: https://open.kattis.com/problems/reduplikation

#include <bits/stdc++.h>
using namespace std;

int main()
{
    ios::sync_with_stdio(false); cin.tie(0); cout.tie(0);
    string s;
    int a;
    cin >> s >> a;
    for(int i = 0; i < a; i++)
        cout << s;

    return 0;
}